packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -0,0 +1,232 @@
+      action: 'linkClick',
+      details: {
+        'event': eventID,
+        'paginate-page-number': page,
+      },
+    });
+  };
+
+  private pageNumbers = () => {
+    const {
+      maxPageListLength,
+      page: currentPage,
+      pages: totalPages,
+      showLastPage,
+    } = this;
+
+    // Make space for "... (last page number)" if not in range of the last page.
+    const showEllipsisAndLastPage =
+      showLastPage && currentPage < totalPages - maxPageListLength + 1;
+
+    const limit = showEllipsisAndLastPage
+      ? maxPageListLength - 2
+      : maxPageListLength;
+
+    let end;
+    let start;
+
+    // If there are more pages returned than the limit to show
+    // cap the upper range at limit + the page number.
+    if (totalPages > limit) {
+      start = currentPage;
+      end = limit + currentPage;
+      // treat the last pages specially
+      if (start >= totalPages - limit + 1) {
+        start = totalPages - limit + 1;
+        end = totalPages + 1;
+      }
+    } else {
+      start = 1;
+      end = totalPages + 1;
+    }
+    return Array.from({ length: end - start }, (_, i) => i + start);
+  };
+
+  private handleKeyDown = (e, pageNumber) => {
+    const keyCode = e.key;
+    if (keyCode === 'Enter' || keyCode === ' ') {
+      e.preventDefault();
+      if (!pageNumber) return;
+      this.handlePageSelect(pageNumber, 'nav-paginate-number');
+    }
+  };
+
+  render() {
+    const { ariaLabelSuffix, page, pages, maxPageListLength, showLastPage } =
+      this;
+
+    if (pages === 1) {
+      return <div />;
+    }
+
+    const renderPages = this.pageNumbers().map(pageNumber => {
+      const pageClass = classnames({
+        'button-active': page === pageNumber,
+        'button-inner': true,
+      });
+
+      return (
+        <li>
+          <button
+            aria-current={page === pageNumber ? 'true' : null}
+            aria-label={`Page ${pageNumber} ${ariaLabelSuffix}`}
+            class={pageClass}
+            onClick={() =>
+              this.handlePageSelect(pageNumber, 'nav-paginate-number')
+            }
+            onKeyDown={e => this.handleKeyDown(e, pageNumber)}
+            type="button"
+          >
+            {pageNumber}
+          </button>
+        </li>
+      );
+    });
+
+    return (
+      <Host role="navigation">
+        <ul class="pagination-prev">
+          {/* START PREV BUTTON */}
+          {this.page > 1 && (
+            <li>
+              <button
+                aria-label={`Previous page ${this.ariaLabelSuffix}`}
+                class="button-prev"
+                onClick={() =>
+                  this.handlePageSelect(this.page - 1, 'nav-paginate-previous')
+                }
+                onKeyDown={e => this.handleKeyDown(e, this.page - 1)}
+                type="button"
+              >
+                Prev
+              </button>
+            </li>
+          )}
+          {/* END PREV BUTTON */}
+        </ul>
```

Comment:
Same question here. In the React component this is rendered as:

```jsx
    <span className="va-pagination-prev">
      <a>Prev</a>
    </span>
```

Reply:
See my previous comment above.